dehydrated

```diff
@@ -842,15 +842,19 @@ command_version() {
   revision="$(cd "${SCRIPTDIR}"; git rev-parse HEAD 2>/dev/null || echo "unknown")"
   echo "GIT-Revision: ${revision}"
   echo ""
-  echo "OS: $(cat /etc/issue | grep -v ^$ | head -n1 | _sed 's/\\(r|n|l) .*//g')"
+  [[ -f /etc/issue ]] && echo "OS: $(cat /etc/issue | grep -v ^$ | head -n1 | _sed 's/\\(r|n|l) .*//g')"
```

Comment:
It's possible to get the equivalent to `/etc/issue` on FreeBSD by running `uname -sr`, which prints both the name of the operating system (`-s`) and the release (`-r`) to stdout. Both these switches are [part of the POSIX `uname`](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/uname.html) and exist on all currently [supported releases](https://www.freebsd.org/security/security.html#sup) (and probably other *BSD systems too).

Reply:
I think /etc/issue is not really something to rely on. eg. on my old gentoo box it contains
'This is \n.\O (\s \m \r) \t
so original line seems to be pretty useless - I just wanted to suppress error from cat...